### PR TITLE
Various changes

### PR DESCRIPTION
**Pull request by hgsommer**
In order to keep my working branch clean, I would like to upstream most of the changes not directly related to my project.
Please point out any changes that might be controversial, then I will move them so a separate PR.

### CODE REVIEW

File: src/alignment/PSSMCalculator.cpp [hunk 1]
```diff
@@ -171,7 +168,6 @@ void PSSMCalculator::computeNeff_M(float *frequency, float *seqWeight, float *Ne
             }
         }
         Neff_M[pos] = (w_M < 0) ? 1.0 : Nlim - (Nlim - 1.0) * MathUtil::fpow2(scale * w_M);
-//        fprintf(stderr,"M  i=%3i  ncol=---  Neff_M=%5.2f  Nlim=%5.2f  w_M=%5.3f  Neff_M=%5.2f\n",pos,Neff_HMM,Nlim,w_M,Neff_M[pos]);
```
Comment (martin-steinegger) [inline src/alignment/PSSMCalculator.cpp]: Please leave comments in the source code. They are used to find bugs.

Comment (hgsommer) [reply src/alignment/PSSMCalculator.cpp]: I have removed the two commits which deleted debug prints.

File: src/commons/Sequence.cpp [hunk 2]
```diff
@@ -300,11 +297,6 @@ void Sequence::mapProfile(const char * profileData, bool mapScores, unsigned int
             if(sumProb > 0.9){
                 MathUtil::NormalizeTo1(&profile[l * PROFILE_AA_SIZE], PROFILE_AA_SIZE);
             }
-            if(nullCnt==PROFILE_AA_SIZE) {
```
Comment (martin-steinegger) [inline src/commons/Sequence.cpp]: I think this was used to mask a profile column. Is this still possible?

Comment (hgsommer) [reply src/commons/Sequence.cpp]: The respective commit message:
"nullCnt was only used to reset a profile column to zero but that could
only happen, if the column contained only zeroes already. In that case,
sumProb will also be zero and the normalize function is not called."

**Pull request merged**